fe1-web/model/network/method/message/data/election/SetupElection.ts

```diff
@@ -48,16 +48,10 @@ export class SetupElection implements MessageData {
     checkTimestampStaleness(msg.start_time);
     if (!msg.end_time) throw new ProtocolError('Undefined \'end_time\' parameter encountered during \'SetupElection\'');
     checkTimestampStaleness(msg.end_time);
-    if (msg.end_time < msg.created_at) throw new ProtocolError('Invalid timestamp encountered: \'end\' parameter smaller than \'creation\'');
+    if (msg.start_time < msg.created_at) throw new ProtocolError('Invalid timestamp encountered: \'start\' parameter smaller than \'created_at\'');
+    this.start_time = msg.start_time;
```

Comment:
you might want to use `{ ... }` around the if body

Reply:
Thanks! I just changed that and it resolved most of them.
Now the only code smells are:
- variable name (Need to be that way for clarity with the JSON)
- switch statement with only 1 case (in the future more cases will come, probably in next PR even)
- one severe one about a long ling which I can't see the issue with:
![image](https://user-images.githubusercontent.com/28142119/117359150-4d0edd00-aeb7-11eb-96aa-d82c3fd4dec0.png)